contracts/axelar-gateway/src/testutils.rs

```diff
@@ -53,6 +53,17 @@ pub fn setup_gateway<'a>(
     (signer_set, client)
 }
 
+pub fn setup_env<'a>(
```

Comment:
This isn't a util being exported to other crates. Define this in `tests/util.rs`instead

Reply:
set it up as `utils/mod.rs` to prevent it from being interpreted as a test file (https://doc.rust-lang.org/rust-by-example/testing/integration_testing.html)

> Each Rust source file in the tests directory is compiled as a separate crate. In order to share some code between integration tests we can make a module with public functions, importing and using it within tests.